pyfarm/agent/etc/agent.yml

```diff
@@ -63,6 +70,7 @@ agent_api_port: 50000
 # The location where the the agent should save its own
 # logging output to.
 agent_log: $agent_logs_root/agent.log
+agent_log: $agent_logs_root/supervisor.log
```

Comment:
Hmm.  Could you explain why this should be supervisor.log instead of agent.log?  The agent itself could be writing to this location and while the supervisor could have a log too I assume we'd want either a combined log file or a separate path for each.

In either case, the log file is not currently implemented....need to fix that...